packages/vue/docs/contributing/coding-guidelines.md

```diff
@@ -266,6 +266,58 @@ A **safe** set of properties to customize are: `font-size`, `text-align`, `color
 Try to make components as customizable as possible **without complicating** them. Think about the parts that are usually customized and allow simple way to change their look.
 :::
 
+## Deprecation guidelines
+
+StorefrontUI cares about providing new features and keeping up-to-date with new best practices. Sometimes, it can be necessary to make some changes in APIs or prepare new features to stay current. To make this process easier for everyone, we deprecate APIs and features for a period before actual removal, by adding proper info about deprecation in the next major version.
```

Comment:
```suggestion
StorefrontUI cares about providing new features and keeping up-to-date with new best practices. Sometimes it can become necessary to make changes to APIs or to prepare new features in order to stay current. To make this process easier for everyone, we deprecate APIs and features for a period of time before the actual removal by adding proper information about deprecations in the next major version.
```